code/game/objects/items/weapons/storage/pouches.dm

```diff
@@ -58,26 +58,46 @@
 
 /obj/item/weapon/storage/pouch/ammo
 	name = "storage pouch (ammo)"
-	desc = "This storage pouch can be used to provide some additional storage for quick access. Can only hold ammunition and cells."
+	desc = "This storage pouch can be used to provide some additional storage for quick access. Can only hold ammunition, cells, explosives, and grenades."
 	icon_state = "ammo"
 	max_storage_space = INVENTORY_POUCH_SPACE
-	can_hold = list(/obj/item/ammo_magazine, /obj/item/ammo_casing, /obj/item/weapon/cell/device/weapon)
+	can_hold = list(/obj/item/ammo_magazine, /obj/item/ammo_casing, /obj/item/weapon/cell/device/weapon, /obj/item/weapon/grenade, /obj/item/weapon/plastique) //Vorestation Add - make it more useful for non-sec/explo
 
 /obj/item/weapon/storage/pouch/eng_tool
 	name = "storage pouch (tools)"
 	desc = "This storage pouch can be used to provide some additional storage for quick access. Can only hold tools."
 	icon_state = "engineering_tool"
 	max_storage_space = INVENTORY_POUCH_SPACE
 	can_hold = list(
-		/obj/item/weapon/tool,
+		/obj/item/weapon/tool/crowbar,
+		/obj/item/weapon/tool/screwdriver,
 		/obj/item/weapon/weldingtool,
+		/obj/item/weapon/tool/wirecutters,
+		/obj/item/weapon/tool/wrench,
 		/obj/item/device/multitool,
 		/obj/item/device/flashlight,
+		/obj/item/weapon/cell/device,
+		/obj/item/stack/cable_coil,
 		/obj/item/device/t_scanner,
 		/obj/item/device/analyzer,
 		/obj/item/clothing/glasses,
 		/obj/item/clothing/gloves,
-	)
+		/obj/item/device/pda,
+		/obj/item/device/megaphone,
+		/obj/item/taperoll,
+		/obj/item/device/radio/headset,
+		/obj/item/device/robotanalyzer,
+		/obj/item/weapon/material/minihoe,
+		/obj/item/weapon/material/knife/machete/hatchet,
+		/obj/item/device/analyzer/plant_analyzer,
+		/obj/item/weapon/extinguisher/mini,
+		/obj/item/weapon/tape_roll,
+		/obj/item/device/integrated_electronics/wirer,
+		/obj/item/device/integrated_electronics/debugger, 
+		/obj/item/weapon/shovel/spade, 
```

Comment:
Pretty sure this is itemsize large which is way too big.

Reply:
Also yeah medical pouch probably draws too fast for being a medium. I'm thinking I will make it small size and keep fast draw so its stuff like "I can put my health analyzer in here and and also have my pill bottle/auto injector in somewhere actually accessible". Or some other "I would put it in my pocket if I didn't need better things there" since that's vaguely the idea of pouches. Admittedly with the specialized ones I didn't feel too bad with giving them nice potential because the basic medium one is so much better in every way and the large size one is even better than that.

Also yeah the BS pouches are BS, full backpack carry size in your pocket with only a normal size item restriction. I didn't touch the balance on those because they're too good and I don't know how to fix that other than cut the storage size down.

> Pretty sure this is itemsize large which is way too big.

https://github.com/VOREStation/VOREStation/blob/5d9f0462c19c2b7fb53bab24ef4d54b87f455ba6/code/modules/mining/mine_items.dm#L171 its this one, which has ITEMSIZE_SMALL. You may be thinking of the normal/wood shovels that are normal size.

Reply:
Ahhh.

TBH, cutting the storage size down would be enough. BS Backpacks are, iirc, roughly twice the space of a regular backpack (might be 2.5x the space, I forgot exactly), so bluespace pouches should be INVENTORY_POUCH_SPACE*2, which would give it 8 slots which is slightly more than a box which is pretty big.

Reply:
I dunno, I think I'll leave it alone for now since it was outside the scope I intended for this PR.

Medical stuff got updated, deleted a slot so its 1 normal 1 small or 3 smalls with the same quick draw, BUT, the quick draw is slow enough that you need to briefly stop to pull stuff out. Also removed the defib as an option and added lollipops.